cmd/traffic/cmd/agent/client.go

```diff
@@ -65,7 +66,12 @@ func TalkToManager(ctx context.Context, address string, info *rpc.AgentInfo, sta
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
-	conn, err := grpc.DialContext(ctx, address, grpc.WithTransportCredentials(insecure.NewCredentials()), grpc.WithBlock())
+	conn, err := grpc.DialContext(ctx, address,
```

Comment:
does this not need to be wrapped in with `if config.AgentConfig().TracingPort != 0 { ... }`?

Reply:
nah it's actually alright; if the config disables tracing the headers will get added but they're totally no-op, traces will just be discarded, i'm pretty sure there's no impact